src/main/java/model/lotto/Lottos.java

```diff
@@ -0,0 +1,46 @@
+package model.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import model.lottonumber.LottoNumbers;
+import model.result.LottoResult;
+import model.winningnumber.WinningLottoNumberDTO;
+import strategy.LottoNumbersGenerationStrategy;
+
+public class Lottos {
+	private List<Lotto> lottoStorage;
+
+	public Lottos(LottoCount manualLottoCount, LottoNumbersGenerationStrategy manualStrategy,
+		LottoCount automaticLottoCount,
+		LottoNumbersGenerationStrategy automaticStrategy) {
+		lottoStorage = Stream.concat(makeLottos(manualLottoCount, manualStrategy).stream(),
+			makeLottos(automaticLottoCount, automaticStrategy).stream()).collect(
+			Collectors.toList());
```

Comment:
`addAll`만으로도 충분히 가능하지 않을까요?

```suggestion
        lottoStorage = makeLottos(manualLottoCount, manualStrategy);
        lottoStorage.addAll(makeLottos(automaticLottoCount, automaticStrategy));
```